docs/root/start/building.rst

```diff
@@ -33,7 +33,9 @@ as the new tcmalloc code is not guaranteed to compile with lower versions of Cla
 Windows Target Requirements
```

Comment:
i wonder whether we should remove this section given we cant guarantee that it will build

Reply:
We need to keep is somewhere for the record, in case someone wants to come in and resurrect Windows build. With the note I think the expectations of Windows build state are clear.